### DISCUSSION TITLE
How to read driver station boot sequence with HDMI

### DISCUSSION THREAD

**Discussion by jds-designs** [gizmo-platform/gizmo#34] [Q&A] [answered]
I just got my hands on a mini HDMI cable, and I'm trying to use it to view the boot sequence of a driver station. However, when I plug it into my computer, my compter doesn't detect anything plugged in. I got the same result with two different driver stations. Are there additional steps I need to do to see the boot sequence display?

Thanks!

**Comment by barulicm** [comment]
To be clear, the HDMI cable is a display cable meant to connect your driver station to a monitor or TV. When connected to a display, the driver station will show its boot sequence like any other computer would.
Is this how you're using your cable?

**Reply by jds-designs** [reply]
Thanks for the clarification. I did also try a driver station with my TV, but it also didn't display anything, even though I set the input to the correct port. (It is a smart TV, I don't know if that would cause any problems.)

**Reply by plurt97** [reply]
FWIW, the Driver Station HDMI port is blocked slightly by the red casing.  We've had to jiggle/place/tilt/fanagle things to make it connect.  We happened to find one micro to hdmi cable that fit somewhat snugly and even then we had to make sure it was plugged in.  So, make sure it's securely plugged in (I've heard of some folks having to remove the casing and plug the hdmi directly into the hardware, outside of the case in order to get a good connection.)

**Reply by the-maldridge** [reply, marked as answer]
The boot firmware that the driver's station uses is fairly simplistic.  It does not have hotplug support, so make sure you have the monitor/TV/capture device plugged in before you power it on.  The case can interfere as @plurt97 points out, so also make sure that the connector is as firmly seated as you can do.

If you want to view the HDMI feed on a laptop, adapters [like this](https://www.amazon.com/gp/product/B0DG2LLJX3) exist which let you view an HDMI source as though it were a camera on the computer.
